### ISSUE TITLE
How was the average of ten times obtained

### ISSUE THREAD

**Issue by X-K11** [Florian-BACHO/FDFA#1]
The paper mentions taking the average of ten results as the final result. How was the ten results obtained? The training and testing sets in the code are fixed, and if other parameters are not changed, the results obtained each time should be consistent

**Comment by Florian-BACHO** [comment]
We ran each experiment ten times with different random seeds. Since the training set is randomly shuffled, we expect slight variations in the resulting metrics. We reported the average and standard deviation of the best performance across all runs.
